tests/e2e/features/artifacts/assetsPage/artifacts-emptyWallet.feature

```diff
@@ -16,4 +16,22 @@ Feature: Artifacts - UI
     Then Element with "href and text" "'/transaction/zksync/era/receive' and 'Add funds'" should be "visible"
     Then Element with "href and text" "'/transaction/zksync/era/receive' and 'Add funds'" should be "clickable"
     Then Element with "href and text" "'/balances' and 'View all'" should be "visible"
-    Then Element with "href and text" "'/balances' and 'View all'" should be "clickable"
\ No newline at end of file
+    Then Element with "href and text" "'/balances' and 'View all'" should be "clickable"
+
+  @tokens @emptyWallet
```

Comment:
This part is to use a solid test scenario for all cases when we need to check zero balance tokens on the wallet.  And also to use @emptyWallet tag.